backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/cfg/ElasticsearchIndexSettings.java

```diff
@@ -57,8 +57,8 @@ private ElasticsearchIndexSettings() {
 	 * <p>
 	 * Expects a string representing the path to a file in the classpath, encoded in UTF-8. The file must contain index settings expressed in JSON format, with the exact same syntax as expected by the Elasticsearch server under the "settings" property <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-create-index.html#create-index-settings">when creating an index</a>. For example, if the file content is <code>{"index.codec": "best_compression"}</code>, it will set {@literal  index.codec} to {@literal best_compression}.
 	 * <p>
-	 * Note that the settings generated by Hibernate Search won't be applied.
-	 * This means for instance that any analyzer defined by the {@link #ANALYSIS_CONFIGURER} will be ignored.
+	 * Note that the settings generated by Hibernate Search will be overridden in case of conflict of some definitions.
+	 * This means for instance that any analyzer defined by the {@link #ANALYSIS_CONFIGURER} could be overridden (or not) by the provided custom settings.
```

Comment:
I understood this as "maybe it will work, maybe it won't".

Maybe this will be less ambiguous:

```suggestion
 	 * For instance, if an analyzer "myAnalyzer" is defined by the {@link #ANALYSIS_CONFIGURER} <strong>and</strong> this settings file, the definition from the settings file will win. If it is only defined in either the analysis configurer or the settings file, but not both, it will be preserved as-is.
```